lib/new_relic/agent/instrumentation/grpc/client/instrumentation.rb

```diff
@@ -13,26 +13,48 @@ module GRPC
         module Client
           include NewRelic::Agent::Instrumentation::GRPC::Helper
 
-          def issue_request_with_tracing(method, requests, marshal, unmarshal,
+          def issue_request_with_tracing(grpc_type, method, requests, marshal, unmarshal,
             deadline:, return_op:, parent:, credentials:, metadata:)
             return yield unless trace_with_newrelic?
 
             segment = request_segment(method)
             request_wrapper = NewRelic::Agent::Instrumentation::GRPC::Client::RequestWrapper.new(@host)
             segment.add_request_headers(request_wrapper)
             metadata.merge!(metadata, request_wrapper.instance_variable_get(:@newrelic_metadata))
+            grpc_message = nil
+            grpc_status = 0
```

Comment:
- For gRPC clients, report the "grpc_message" and "grpc_status". In the event of
  an error, these will match the gRPC status "code" and "number", respectively
  as defined in the [gRPC status codes list](https://grpc.github.io/grpc/core/md_doc_statuscodes.html).
  In the event of a success, the message will be `nil` and the status will be `0`.